cluster/examples/kubernetes/monitoring/tectonic-service-monitor.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
It seems that the difference between these and https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/monitoring/service-monitor.yaml is that you are specifying a port. Is this required or can you use the example i posted for tectonic?

Reply:
I will retry this!

Reply:
This is absolutely possible, I thought first the port was needed but upon the current setup I have it works just fine also without the port
Changes applied in last commit

Reply:
@hacor Is there any big difference between this file and https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/monitoring/service-monitor.yaml if not this file here should be removed.

Reply:
Only differences I find for this moment are:
- the `targetPort` (which I thought was necessary, but I'll retry this)
- the versioning: `v1` for Tectonic and `v1alpha` for Rook

I'll try wether the differences really matter.

Reply:
The current ServiceMonitor (https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/monitoring/service-monitor.yaml) has been updated to use `v1`.
I'm using multiple ServiceMonitors without `targetPort` so I guess it isn't "required" as written in the API docs of prometheus-operator https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint.

Please remove this file and point your docs to the existing `service-monitor.yaml` as it seems to be redundant.